### PR TITLE
tests: update govcloud to use roles and the new aws govcloud account

### PR DESCRIPTION
**Pull request by cpanato**
fixes: INST-921

### CODE REVIEW

File: tests/rspec/spec/govcloud/vpc_internal_spec.rb [hunk 1]
```diff
@@ -13,7 +13,10 @@
   )
 
   before(:all) do
-    @ssh_key = ENV['TF_VAR_tectonic_govcloud_ssh_key'] || AwsSupport.create_aws_key_pairs('us-gov-west-1')
+    @role_credentials = nil
```
Comment (enxebre) [inline tests/rspec/spec/govcloud/vpc_internal_spec.rb]: curious why are we initialising to nil

Comment (cpanato) [reply tests/rspec/spec/govcloud/vpc_internal_spec.rb]: because if we don't use that, it will pass nil to the functions that call aws

Comment (cpanato) [reply tests/rspec/spec/govcloud/vpc_internal_spec.rb]: then we can support both cases: with roles and without role

**Comment by cpanato** [comment]
adding other platforms just to test the changes in the Jenkins file.
retest this please.

**Pull request merged**